parity/configuration.rs

```diff
@@ -564,10 +564,10 @@ impl Configuration {
 		let dapps_path = replace_home(&self.args.flag_dapps_path);
 		let signer_path = replace_home(&self.args.flag_signer_path);
 
-		if self.args.flag_geth {
-			let geth_path = path::ethereum::default();
-			::std::fs::create_dir_all(geth_path.as_path()).unwrap_or_else(
-				|e| warn!("Failed to create '{}' for geth mode: {}", &geth_path.to_str().unwrap(), e));
+		if self.args.flag_geth  && !cfg!(windows) {
```

Comment:
no need to create directory for windows pipe, it is in no way related to the file system